inexor/engine/rendergl.cpp

```diff
@@ -581,6 +581,7 @@ void cef_resize(int width, int height)
         // TODO: not fully working
         spdlog::get("global")->info("Update Inexor User Interface Screen Size: {0}x{1}", width, height);
         cef_app->GetHudLayer()->Resize(0, 0, width, screen_manager.screenh);
+        cef_app->GetConsoleLayer()->Resize(0, 0, width, screen_manager.screenh);
```

Comment:
:+1: 

Sidenote: why is this code-part so inconsistent? Oo

local and global vars mixed.. just asking, working anyways probably.